workshops/portfolio/README.md

```diff
@@ -248,7 +246,7 @@ side, we see that our original text is still there:
 
 > ![](img/scroll_down.gif)
 
-Ok. We need to make the picture smaller
+Ok. Let's make the picture smaller
```

Comment:
Pretty sure this should be "lets"

http://public.wsu.edu/~brians/errors/lets.html